apps/paper-tester/ios/papertester/Info.plist

```diff
@@ -55,7 +55,7 @@
 	<key>NSMicrophoneUsageDescription</key>
 	<string>Allow $(PRODUCT_NAME) to access your microphone</string>
 	<key>RCTNewArchEnabled</key>
-	<false/>
+	<true/>
```

Comment:
Should we consider removing or renaming this project? We won't test on the old arch anymore?